### DISCUSSION TITLE
Windows version?

### DISCUSSION THREAD

**Discussion by gbdrbob** [SteamDeckHomebrew/decky-loader#39] [Ideas]
Some games will never be made compatible with steamOS due to developer/publisher intransigence over anti-cheat, and many users will want to dual boot Windows on their Steam Deck so they can run game pass games locally.

A Windows version of Plugin Loader would therefore be useful as it would help with porting of steamOS features like the quick access menu etc over to Windows. Eventually, Windows on the Deck could have a similar user experience to SteamOS on the Deck.

**Comment by TrainDoctor** [comment]
Valve has no plans to port quick access to windows.
Plugin Loader relies on SteamOS being installed in order to operate. If you would like to implement something similar to plugin loader under windows feel free, but considering the amount of time and effort that such a thing would take I can't say I'm at all interested in the idea.

**Comment by marios8543** [comment]
No, this is unlikely to ever happen.

**Comment by Filip-Anicic** [comment]
You can get the steam deck UI on windows by changing some settings in steam, but idk if the deck overlay works. If it does, you can fork the PluginLoader and update the paths Windows installation folder of Steam instead of Linux one, though that doesn't solve not having any plugins that run compatible commands and paths with Windows.

In my opinion, I'd just stream with steam link from my PC to my steam deck if the game can't run on Proton/Linux. Would save a lot of time.

**Reply by milroneth** [reply]
The Deck overlay kinda works (it doesn't actually overlay your games, but it opens inside the Steam window). Same for the QAM, you can open it in Windows just fine, but the thing is that it's quite useless since almost none of its settings are available on Windows.

For the plugins, it's even worse. They are using Linux specific functions which would not work on Windows at all.

So, a completely different plugin system with its own separate plugin set, would be possible if some developers decided to make it one day, after Valve makes the new Steam client properly work on Windows, which might be... whenever, but this one is impossible to work on Windows.

**Comment by TrainDoctor** [comment]
The team has no plans to work on any windows version of plugin loader at the moment, as the focus of Plugin Loader is expanding the capabilities of the actual Steam Deck hardware in gaming mode. Windows support (outside of support for developers) is not planned and will not happen (aka pull requests for such support will be closed as wontfix). Wanted to clarify this stance before closing this discussion.
